src/pages/SubmissionConfig.test.js

```diff
@@ -2163,6 +2236,102 @@ describe('SubmissionConfig submitting a run set', () => {
       })
     )
   })
+
+  it('should call POST /run_sets endpoint with expected parameters after outputs are set to default', async () => {
```

Comment:
How does this kind of test separation sound? Do you think we'd cover our bases or would there still potentially be gaps?
- Does the SubmissionConfig page use the state set by the `OutputsTable` appropriately when POSTing (1 cross-component test)
- Does the `OutputsTable` set the state that's going to be used by the SubmissionConfig page appropriately? (many component-internal tests?)

Reply:
Yes, I agree this would be a good candidate for the separation we discussed yesterday. I wrote these tests prior to the discussion but it should be pretty easy to go in and shift them.